gloss-examples/picture/Flake/Main.hs

```diff
@@ -1,7 +1,8 @@
-
 -- | Snowflake Fractal.
 --      Based on ANUPlot code by Clem Baker-Finch.
 --
+module Main where
```

Comment:
I'm not 100% sure why, but modern `cabal` (although not `stack`) was complaining if there wasn't an explicit `module` declaration here.